packages/react-native-platform-override/src/Hash.ts

```diff
@@ -11,29 +11,52 @@ import {normalizePath, unixPath} from './PathUtils';
 import FileRepository from './FileRepository';
 import isutf8 from 'isutf8';
 
+export type HashOpts = {
+  /**
+   * What differences between files should we be insensitive to when generating
+   * a hash? Defaults to line-ending insensitivity
+   */
+  insensitivity?: 'none' | 'line-ending' | 'whitespace';
+};
+
+/**
+ * Normalize content according to insensitivty rules
+ */
+function normalizeContent(
```

Comment:
Since this is for hashing, would `canonicalize` make more sense than `normalize`?

Reply:
I needed to look up the definition of `canonical` to remember what it meant 😂. I think my own preference would be normalize.